support-frontend/webpack.common.js

```diff
@@ -41,6 +41,25 @@ const cssLoaders = [{
   },
 }];
 
+// Hide mini-css-extract-plugin spam logs
```

Comment:
Was this causing a real problem?

Reply:
not a big problem, just `mini-css` logs hid any useful build messages and occasionally showed false positive errors